pom.xml

```diff
@@ -1250,6 +1249,11 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-pmd-plugin</artifactId>
```

Comment:
I just moved it next to other code analysis tool because it seems more convenient to have them next to each other. The plugin is already in the `build.plugins` section. It's not configured to do anything afaik. So, the logic should remain the same. I'll use it in #3020 where configuration is handled in `pluginManagement`.

Reply:
Ok, makes sense.